30_regexp_lab/relab.txt

```diff
@@ -1,17 +1,18 @@
 Use grep to print only the lines from "input" for which:
 
 1. There is a binary number in the line
-
+grep -E [10] input.txt
```

Comment:
it's a good start, but not enough by itself, consider this:
```
vitali@Tux:~/tmp$ echo 943423400112353 | grep [10]
943423400112353
```
we need to match words, there is ```\b```[ that will match words ](http://www.regular-expressions.info/wordboundaries.html)

try:
```grep -E "\b[10]+\b" # the + means match this more then once```

also note that ```-E``` is needed only when using extended regex, it was redundant with only ```[10]```